growth/too/tasks/growthdb_cgi.py

```diff
@@ -0,0 +1,124 @@
+import requests
+import json
+import numpy as np
+import astropy.units as u
+import datetime
+from celery.task import PeriodicTask
+from celery.utils.log import get_task_logger
+from celery.local import PromiseProxy
+import celery
+
+from .. import models
+
+"""
+Reminder for the relevant program names:
+decam_programidx=program_dict['DECAM GW Followup']
+em_gw_programidx=program_dict['Electromagnetic Counterparts to Gravitational Waves']
+fermi_programidx=program_dict['Afterglows of Fermi Gamma Ray Bursts']
+neutrino_programidx=program_dict['Electromagnetic Counterparts to Neutrinos']
+"""
+
+
+def get_programidx(program_name):
+    """ Given a program name, it returns the programidx """
+
+    r = requests.post('http://skipper.caltech.edu:8080/cgi-bin/growth/list_programs.cgi')
```

Comment:
Please add `r.raise_for_status()` so that we get an exception for any HTTP status code other than 200.